apps/admin/frontend/src/screens/print_test_deck_screen.tsx

```diff
@@ -454,37 +551,73 @@ export function PrintTestDeckScreen(): JSX.Element {
             <li>One overvoted hand-marked test ballot.</li>
           </ol>
           <HorizontalRule />
-        </Prose>
-        <p>
-          <PrintButton
-            print={() => printLetterComponentsOfLogicAndAccuracyPackage('all')}
-            useDefaultProgressModal={false}
-            fullWidth
-          >
-            <strong>Print Packages for All Precincts</strong>
-          </PrintButton>
-        </p>
-        <ButtonList>
-          {[...election.precincts]
-            .sort((a, b) =>
-              a.name.localeCompare(b.name, undefined, {
-                ignorePunctuation: true,
-              })
-            )
-            .map((p) => (
-              <PrintButton
-                key={p.id}
-                print={() =>
-                  printLetterComponentsOfLogicAndAccuracyPackage(p.id)
-                }
-                useDefaultProgressModal={false}
+          <ButtonList>
+            <PrintButton
+              print={() =>
+                printLetterComponentsOfLogicAndAccuracyPackage('all')
+              }
+              useDefaultProgressModal={false}
+              fullWidth
+            >
+              <strong>Print Packages for All Precincts</strong>
+            </PrintButton>
+            {window.kiosk && (
+              <Button
+                onPress={() => onClickSaveLogicAndAccuracyPackageToPdf('all')}
                 fullWidth
               >
-                {p.name}
-              </PrintButton>
-            ))}
-        </ButtonList>
+                Save Packages for All Precincts as PDF
+              </Button>
+            )}
+          </ButtonList>
+          <HorizontalRule />
+          <ButtonList>
+            {[...election.precincts]
+              .sort((a, b) =>
+                a.name.localeCompare(b.name, undefined, {
+                  ignorePunctuation: true,
+                })
+              )
+              .map((p) => (
+                <React.Fragment key={p.id}>
+                  <PrintButton
+                    print={() =>
+                      printLetterComponentsOfLogicAndAccuracyPackage(p.id)
+                    }
+                    useDefaultProgressModal={false}
+                    fullWidth
+                  >
+                    Print {p.name}
+                  </PrintButton>
+                  {window.kiosk && (
+                    <Button
+                      onPress={() =>
+                        onClickSaveLogicAndAccuracyPackageToPdf(p.id)
+                      }
+                      fullWidth
+                    >
+                      Save {p.name} to PDF
+                    </Button>
+                  )}
+                </React.Fragment>
```

Comment:
Just sanity checking, does this page look okay when there are enough precincts to span multiple rows?

Ideally, there's more space between the precinct rows than the two buttons for a precinct, e.g.

```
[ Print Precinct 1       ] [ Print Precinct 2       ] [ Print Precinct 3       ]
[ Save Precinct 1 to PDF ] [ Save Precinct 2 to PDF ] [ Save Precinct 3 to PDF ]
                        <-- More space here
[ Print Precinct 4       ]
[ Save Precinct 4 to PDF ]
```

Excuse my ASCII art mock 😅

Reply:
Good call, here's the before and after.

![Screenshot 2023-03-07 at 2 38 49 PM](https://user-images.githubusercontent.com/1060688/223573434-beb3d311-1eaf-4cdc-a46c-872f00501d24.png)
![Screenshot 2023-03-07 at 2 56 22 PM](https://user-images.githubusercontent.com/1060688/223573440-11bcaf72-2984-49b5-8745-d653d0069ad2.png)


Curious what our philosophy on spacing is - I don't see a lot of CSS or design system spacing (eg. `.margin-bottom-x` classes) and have instead seen actual HTML elements used for spacing. Is there a best practice for this? Did I miss a CSS file somewhere that I should edit?

Reply:
Yeah great q! Our styles are managed using [styled-components](https://styled-components.com) rather than standalone CSS files. Consistency comes from shared container components, which dictate how children are spaced. The prime example of this is the `Prose` component, which you'll see throughout code:

https://github.com/votingworks/vxsuite/blob/750fd899443d7fbc1304b4e4668bf144a7d37866/libs/ui/src/prose.tsx#L30-L103

Providing another example, there's also `ButtonList`:

https://github.com/votingworks/vxsuite/blob/750fd899443d7fbc1304b4e4668bf144a7d37866/libs/ui/src/button_list.tsx#L3-L15

There are of course still rare cases where additional styles have to be added to components, in which case shared spacing variables would make sense. We don't have any yet as far as I know.